### PR TITLE
Bug 1315115 - Add toast to "Switch to tab" to "Open in Tab". r?farhan

### PR DESCRIPTION
**Pull request by ncalexan**
Open questions:
* Is there a danger keeping the reference to the Tab like this?  Can it disappear beneath me?
* Should I prepare and display the toast before waiting for the scroll controller to show the toolbars?  (I didn't in order to minimize the added code.)
* Is the ButtonToast display logic correct?  Is this registering the toast appropriately?
* Are the strings localized correctly?
* How hard is it to add tests for this functionality?

### CODE REVIEW

**Comment by ncalexan** [comment]
@farhanpatel how do I request review these days?  Do I do that in Bugzilla?

File: Client/Frontend/Browser/BrowserViewController.swift [hunk 1]
```diff
@@ -2958,16 +2962,29 @@ extension BrowserViewController: ContextMenuHelperDelegate {
                 let newTabTitle = NSLocalizedString("Open in New Tab", comment: "Context menu item for opening a link in a new tab")
                 let openNewTabAction =  UIAlertAction(title: newTabTitle, style: UIAlertActionStyle.Default) { (action: UIAlertAction) in
                     self.scrollController.showToolbars(animated: !self.scrollController.toolbarsShowing, completion: { _ in
-                        self.tabManager.addTab(NSURLRequest(URL: url), afterTab: currentTab)
+                        let tab = self.tabManager.addTab(NSURLRequest(URL: url), afterTab: currentTab)
+                        let toast = ButtonToast(labelText: NSLocalizedString("New Tab opened", comment: "Button toast label for switching to fresh New Tab"), buttonText: NSLocalizedString("Switch", comment: "Button toast button for switching to fresh New Tab"), completion: { buttonPressed in
```
Comment (farhanpatel) [inline Client/Frontend/Browser/BrowserViewController.swift]: The localized strings should be in Strings.swift.

**Comment by farhanpatel** [comment]
You can check if the Top Tabs is visible by checking if toptabsController exists or not.

File: Client/Frontend/Browser/BrowserViewController.swift (hunk 1 above)
Comment (ncalexan) [inline Client/Frontend/Browser/BrowserViewController.swift]: > I don't think this UI should show on the iPad though.

I concur; will update when I get a minute.

**Comment by ncalexan** [comment]
@farhanpatel back to you.

**Comment by farhanpatel** [comment]
I've refactored for swift 3 in the above PR. Closing this one.

**Pull request closed without merging**